screen-orientation/lock-unlock-check.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script>
+  import { getOppositeOrientation } from "/screen-orientation/resources/orientation-utils.js";
```

Comment:
This is supposed to throw an early SyntaxError, as static `import` only works within `<script type=module>`, so the rest of the `<script>` is essentially ignored. This seems like a bug? cc @leszekswirski

Reply:
Filed here https://github.com/web-platform-tests/wpt/issues/17547

Reply:
Thanks @mathiasbynens! how weird... I thought we had these working but you are absolutely right that these would be syntax errors :/